packages/react/src/themes/teams-dark/siteVariables.ts

```diff
@@ -32,6 +32,12 @@ export const red = '#d74654' // no mapping color
 export const red08 = '#4f232b' // no mapping color
 export const green04 = naturalColors.lightGreen[900]
 
+//
+// BORDER STYLES
+//
+export const focusInnerBorderColor = colors.black
```

Comment:
these don't need to swap for dark theme. the reason we do a double border of white and black is so we can keep the exact same style for both themes

Reply:
done